msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Together Mode in Teams
+description: Work with Together Mode 
+ms.topic: conceptual
+---
+
+# Together Mode in Teams
+
+Teams Together Mode provides a simple, innovative, and user-friendly meeting environment. It digitally combines participants into a single virtual space instead of being shown separately in boxes. It creates your virtual image and places the image in a virtual environment where you can easily interact with your peers. It makes meetings more engaging in the auditorium view.
```

Comment:
Please reword this paragraph: 

Microsoft Teams Together Mode provides an immersive and engaging meeting environment that brings people together and encourages them to turn on their video. It digitally combines participants into a single virtual scene and places their video streams in pre-determined spots designed and fixed by the scene creator.